### DISCUSSION TITLE
Support hysteria

### DISCUSSION THREAD

**Discussion by ghost** [zzzgydi/clash-verge#59] [Ideas]
New repository to create cross-platform GUI client of [hysteria](https://github.com/hynetwork/hysteria)

It seems that clash-core won't support hysteria

_Originally posted by @Dreamacro in https://github.com/Dreamacro/clash/issues/1863#issuecomment-1094498033_

**Comment by zzzgydi** [comment]
这个是要怎么支持的呢

**Reply by ghost** [reply]
Create a new repository called hysteria-verge only using hysteria binary core.

**Comment by txyyh** [comment]
Clash.Meta [has support hysteria](https://github.com/MetaCubeX/Clash.Meta/releases/tag/v1.12.0)
You can upgrade core manually by simply replace clash core file or wait for clash-verge update
